x-pack/legacy/plugins/ml/public/application/management/index.ts

```diff
@@ -10,22 +10,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { npSetup } from 'ui/new_platform';
-import { management } from 'ui/management';
+import { npSetup, npStart } from 'ui/new_platform';
```

Comment:
i don't think we should be importing anything from `ui/new_platform` anymore.
I wonder if this whole file needs to go and for the management section registration needs to happen in our root `plugin.ts` file?

Reply:
I'm not sure this can be done as part of this PR. The only reference that ties into this code is `managementSections: ['plugins/ml/application/management']` in `plugins/ml/index.ts` and I can't see how I'd pass on something to here. If we can think of a solution I'm happy to do it in this PR, however, the original scope here was only to migrate away from angular :). For transforms, the "cut-over" PR was not just moving files but it also included updating the `plugins/transfrom/index.ts` file to work again after the move, maybe we need to do it this way.